packages/ui/src/memberships/components/MemberProfile/MemberDetails.tsx

```diff
@@ -22,6 +24,12 @@ type Props = { member: Member }
 export const MemberDetails = React.memo(({ member }: Props) => {
   const { member: memberDetails, isLoading } = useMember(member.id)
 
+  const myMemberships = useMyMemberships()
+  const isMyMembership = useMemo(() => myMemberships.members.some(({ id }) => id === member.id), [
+    member.id,
+    myMemberships,
+  ])
```

Comment:
WDYT about exposing `isMyMember()` in the `useMyMemberships()` hook? I think that we potentially have other places that require a similar check.

I'm thinking about something like this:

```typescript
const { isMyMembership } = useMyMemberships()

return (
  <>
    {isMyMembership(member.id) ? "It's you!" : "Sorry, I can't do this Dave."}
  </>
)
```

Reply:
Makes sense :ok_hand: